src/stores/spaces/SpaceStore.ts

```diff
@@ -1117,7 +1117,7 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
                 // Don't auto-switch rooms when reacting to a context-switch or for new rooms being created
                 // as this is not helpful and can create loops of rooms/space switching
                 const isSpace = payload.justCreatedOpts?.roomType === RoomType.Space;
-                if ((payload.context_switch || payload.justCreatedOpts)&&!isSpace) break;
+                if ((payload.context_switch || payload.justCreatedOpts) && (!isSpace)) break;
```

Comment:
```suggestion
                if ((payload.context_switch || payload.justCreatedOpts) && !isSpace) break;
```

Redundant parentheses

Reply:
Where to add brackets to make it more readable ? sorry I didn't get your previous comment.